src/Controllers/BatchProcessorController.ts

```diff
@@ -415,19 +431,61 @@ module StreamStats.Controllers {
                     }
 
                     else {
-                        // this.streamStatsAvailable = false;
-                        // this.toaster.pop('error', "No parameters available for this region", "Please select another region");
                     }
                     return paramRaw;
                 }, (error) => {
-                    // console.log('Bad response from the regression service');
-                    // this.streamStatsAvailable = false;
-                    // this.toaster.pop('warning', "StreamStats not available here at this time", "", 5000);
-
                 }).finally(() => {
                 });
         }
 
+        public validateZipFile($files): void {
+            // validate that the file is a .zip
+            if ($files[0].type != "application/x-zip-compressed") {

```

Comment:
I couldn't successfully upload any .zip files, but this fixed it. I think we'll want to validate  Here's a sample file to test
[pointsNC.zip](https://github.com/USGS-WiM/StreamStats/files/11290844/pointsNC.zip)

In the HTML file, we'll want to check for both application/zip and x-zip-compressed attachment types too.


```suggestion
            if ($files[0].type != "application/x-zip-compressed"  && $files[0].type != "application/zip") {

```